content/swift/concepts/methods/methods.md

```diff
@@ -0,0 +1,69 @@
+---
+
+Title: 'Swift Methods'
+
+Description: 'Learn about methods in Swift and how they are used to perform actions and provide functionality
```

Comment:
Typo here, forgot to close this string with a tick

Can you also match this to the beginning definition in the intro paragraph below?